src/neo/Network/P2P/LocalNode.cs

```diff
@@ -31,16 +29,6 @@ public class SendDirectly { public IInventory Inventory; }
         public static readonly uint Nonce;
         public static string UserAgent { get; set; }
 
-        private static LocalNode singleton;
-        public static LocalNode Singleton
-        {
-            get
-            {
-                while (singleton == null) Thread.Sleep(10);
-                return singleton;
-            }
-        }
-
         static LocalNode()
```

Comment:
So how can we get the local node instance in plugins?

Reply:
Why do you need to access `LocalNode` instance in plugins?

Reply:
My goal is to eliminate all singletons. If there is a need to access the `LocalNode` instance, we can pass the instance.

Reply:
There is no way to cast an actor to the underlying object?

Reply:
No way.

Reply:
We can send `this` when we receive a specific message.